CONTRIBUTING.md

```diff
@@ -289,7 +294,7 @@ Had no year associated with it, so it has keywords for its tag `pca-visually-exp
 
 #### Spell checking
 
-Spell checks are run automatically using GitHub actions upon opening a PR for `master` or `staging`.
+Spell checks are [run automatically using GitHub actions](#github-actions-summary) upon opening a PR for `master` or `staging`.
```

Comment:
```suggestion
Spell checks are [run automatically using GitHub actions](#github-actions) upon opening a PR for `master` or `staging`.
```

Same comment here as above.

Reply:
Good catches! I changed the section name but forgot to update the links.